releasenotes/notes/0.10/0-10-release-8c37dadcc1c82fcc.yaml

```diff
@@ -1,66 +1,66 @@
 ---
-prelude: >
+prelude: |
     The Qiskit Aer 0.10 release includes several performance and noise model
-    improvements. Some highlights are
+    improvements. Some highlights are:
+
```

Comment:
Having a space here caused reno build to fail in CI so I had to remove it

Reply:
That was because of the `>`/`|` thing above.